src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Models/Apprentice/DetailsViewModel.cs

```diff
@@ -57,6 +58,10 @@ public class DetailsViewModel : IAuthorizationContextModel
         public bool EmailAddressConfirmedByApprentice { get; set; }
         public bool CanResendInvitation => !string.IsNullOrEmpty(Email) && !EmailAddressConfirmedByApprentice;
         public string DeliveryModel { get; set; }
+        public int? EmploymentPrice { get; set; }
+        public string EmploymentPriceDisplay => EmploymentPrice.HasValue ? EmploymentPrice.Value.ToGdsCostFormat() : string.Empty;
+        public DateTime? EmploymentEndDate { get; set; }
+        public string EmploymentEndDateDisplay => EmploymentEndDate.HasValue ? EmploymentEndDate.Value.ToGdsFormatWithoutDay() : string.Empty;
```

Comment:
I find this simpler to read, what do you think?

```suggestion
        public string EmploymentPriceDisplay => EmploymentPrice?.ToGdsCostFormat() ?? string.Empty;
        public DateTime? EmploymentEndDate { get; set; }
        public string EmploymentEndDateDisplay => EmploymentEndDate?.ToGdsFormatWithoutDay() ?? string.Empty;
```